week2/homework/Program.cs

```diff
@@ -0,0 +1,29 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => (string[] args)=> {
+    
+    string sentence = "the quick brown fox jumps over the lazy dog";
```

Comment:
The code seems a bit misaligned. You can use `Alt+Shift+F` in visual studio code to automatically format it 😊

Reply:
Sure!! Thank you :)